cmd/k8s-bigip-ctlr/main.go

```diff
@@ -112,6 +112,7 @@ var (
 	sslInsecure        *bool
 	trustedCertsCfgmap *string
 	agent              *string
+	logResponseBody    *bool
```

Comment:
logResponseBody - the option "log-response-body" and this param are a bit ambiguous.

Can we have something like, log-bigip-response ??

Reply:
log-as3-response

Reply:
Changed to log-as3-response.